common/config/src/lib.rs

```diff
@@ -190,6 +190,18 @@ impl Config {
             print_settings(&yaml_str, &config_path);
         }
 
+        for (key, value) in std::env::vars() {
+            // Check if the key starts with "LOGDNA_" or "MZ_"
+            if key.starts_with("LOGDNA_") || key.starts_with("MZ_") {
+                // Print the key and its value
+                if key.contains("KEY") || key.contains("PIN") {
+                    info!("{}: [REDACTED]", key);
+                } else {
+                    info!("{}: {}", key, value);
+                }
+            }
+        }
```

Comment:
Printing all the env vars might leak some secrets besides the ones we recognize, and the agent already prints out its config values. Is this just a left over from testing?

Reply:
Requested from IBM support.

Reply:
I am going do it in separate PR. that will not use open list.